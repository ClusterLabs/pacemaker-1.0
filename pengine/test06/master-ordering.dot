digraph "g" {
"apache2:0_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"apache2:0_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"drbd_mysql:0_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"drbd_mysql:0_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"drbd_mysql:0_post_notify_start_0 webcluster01" -> "ms_drbd_mysql_confirmed-post_notify_start_0" [ style = bold]
"drbd_mysql:0_post_notify_start_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"drbd_mysql:0_start_0 webcluster01" -> "ms_drbd_mysql_running_0" [ style = bold]
"drbd_mysql:0_start_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"drbd_www:0_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"drbd_www:0_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"drbd_www:0_post_notify_start_0 webcluster01" -> "ms_drbd_www_confirmed-post_notify_start_0" [ style = bold]
"drbd_www:0_post_notify_start_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"drbd_www:0_start_0 webcluster01" -> "ms_drbd_www_running_0" [ style = bold]
"drbd_www:0_start_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"extip_1_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"extip_1_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"extip_1_monitor_30000 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"extip_1_start_0 webcluster01" -> "extip_1_monitor_30000 webcluster01" [ style = bold]
"extip_1_start_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"extip_2_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"extip_2_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"extip_2_monitor_30000 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"extip_2_start_0 webcluster01" -> "extip_2_monitor_30000 webcluster01" [ style = bold]
"extip_2_start_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"fs_mysql_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"fs_mysql_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"intip_0_main_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"intip_0_main_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"intip_1_master_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"intip_1_master_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"intip_1_master_monitor_30000 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"intip_1_master_start_0 webcluster01" -> "intip_1_master_monitor_30000 webcluster01" [ style = bold]
"intip_1_master_start_0 webcluster01" -> "ms_drbd_mysql_start_0" [ style = bold]
"intip_1_master_start_0 webcluster01" -> "ms_drbd_www_start_0" [ style = bold]
"intip_1_master_start_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"intip_2_slave_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"intip_2_slave_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"intip_2_slave_monitor_30000 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"intip_2_slave_start_0 webcluster01" -> "intip_2_slave_monitor_30000 webcluster01" [ style = bold]
"intip_2_slave_start_0 webcluster01" -> "ms_drbd_mysql_start_0" [ style = bold]
"intip_2_slave_start_0 webcluster01" -> "ms_drbd_www_start_0" [ style = bold]
"intip_2_slave_start_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"ms_drbd_mysql_confirmed-post_notify_start_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_mysql_confirmed-pre_notify_start_0" -> "ms_drbd_mysql_start_0" [ style = bold]
"ms_drbd_mysql_confirmed-pre_notify_start_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_mysql_post_notify_start_0" -> "drbd_mysql:0_post_notify_start_0 webcluster01" [ style = bold]
"ms_drbd_mysql_post_notify_start_0" -> "ms_drbd_mysql_confirmed-post_notify_start_0" [ style = bold]
"ms_drbd_mysql_post_notify_start_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_mysql_pre_notify_start_0" -> "ms_drbd_mysql_confirmed-pre_notify_start_0" [ style = bold]
"ms_drbd_mysql_pre_notify_start_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_mysql_running_0" -> "ms_drbd_mysql_post_notify_start_0" [ style = bold]
"ms_drbd_mysql_running_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_mysql_start_0" -> "drbd_mysql:0_start_0 webcluster01" [ style = bold]
"ms_drbd_mysql_start_0" -> "ms_drbd_mysql_running_0" [ style = bold]
"ms_drbd_mysql_start_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_mysql_stop_0" -> "ms_drbd_mysql_start_0" [ style = bold]
"ms_drbd_mysql_stop_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_www_confirmed-post_notify_start_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_www_confirmed-pre_notify_start_0" -> "ms_drbd_www_start_0" [ style = bold]
"ms_drbd_www_confirmed-pre_notify_start_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_www_post_notify_start_0" -> "drbd_www:0_post_notify_start_0 webcluster01" [ style = bold]
"ms_drbd_www_post_notify_start_0" -> "ms_drbd_www_confirmed-post_notify_start_0" [ style = bold]
"ms_drbd_www_post_notify_start_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_www_pre_notify_start_0" -> "ms_drbd_www_confirmed-pre_notify_start_0" [ style = bold]
"ms_drbd_www_pre_notify_start_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_www_running_0" -> "ms_drbd_www_post_notify_start_0" [ style = bold]
"ms_drbd_www_running_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_www_start_0" -> "drbd_www:0_start_0 webcluster01" [ style = bold]
"ms_drbd_www_start_0" -> "ms_drbd_www_running_0" [ style = bold]
"ms_drbd_www_start_0" [ style=bold color="green" fontcolor="orange"  ]
"ms_drbd_www_stop_0" -> "ms_drbd_www_start_0" [ style = bold]
"ms_drbd_www_stop_0" [ style=bold color="green" fontcolor="orange"  ]
"mysql-proxy:0_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"mysql-proxy:0_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"mysql-server_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"mysql-server_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"ocfs2_www:0_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"ocfs2_www:0_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"ocfs2_www:1_monitor_0 webcluster01" -> "probe_complete webcluster01" [ style = bold]
"ocfs2_www:1_monitor_0 webcluster01" [ style=bold color="green" fontcolor="black"  ]
"probe_complete webcluster01" -> "probe_complete" [ style = bold]
"probe_complete webcluster01" [ style=bold color="green" fontcolor="black"  ]
"probe_complete" -> "drbd_mysql:0_start_0 webcluster01" [ style = bold]
"probe_complete" -> "drbd_www:0_start_0 webcluster01" [ style = bold]
"probe_complete" -> "extip_1_start_0 webcluster01" [ style = bold]
"probe_complete" -> "extip_2_start_0 webcluster01" [ style = bold]
"probe_complete" -> "intip_1_master_start_0 webcluster01" [ style = bold]
"probe_complete" -> "intip_2_slave_start_0 webcluster01" [ style = bold]
"probe_complete" -> "ms_drbd_mysql_start_0" [ style = bold]
"probe_complete" -> "ms_drbd_mysql_stop_0" [ style = bold]
"probe_complete" -> "ms_drbd_www_start_0" [ style = bold]
"probe_complete" -> "ms_drbd_www_stop_0" [ style = bold]
"probe_complete" [ style=bold color="green" fontcolor="orange"  ]
}
