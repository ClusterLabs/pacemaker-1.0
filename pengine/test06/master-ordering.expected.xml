<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="340" num_updates="42004" dc-uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298" have-quorum="true" remote-tls-port="0" validate-with="pacemaker-1.0">
   <configuration>
     <crm_config>
       <cluster_property_set id="default"><nvpair id="nvpair" name="symmetric-cluster" value="true"/><nvpair id="nvpair" name="transition-idle-timeout" value="60"/><nvpair id="nvpair" name="default-resource-stickiness" value="0"/><nvpair id="nvpair" name="short-resource-names" value="true"/><nvpair id="nvpair" name="no-quorum-policy" value="stop"/><nvpair id="nvpair" name="stop-orphan-resources" value="false"/><nvpair id="nvpair" name="stop-orphan-actions" value="true"/><nvpair id="nvpair" name="remove-after-stop" value="false"/><nvpair id="nvpair" name="is-managed-default" value="true"/><nvpair id="nvpair" name="stonith-enabled" value="false"/><nvpair id="nvpair" name="stonith-action" value="reboot"/></cluster_property_set>
     </crm_config>
     <nodes>
       <node id="49e81295-8e2f-4aeb-98f3-a14de6f62298" uname="webcluster01" type="normal"/>
       <node id="917954cd-0285-4fcd-9cd2-671736c4de66" uname="webcluster02" type="normal"/>
     </nodes>
     <resources>
       <primitive id="extip_1" class="ocf" type="IPaddr2" provider="heartbeat"><meta_attributes id="primitive-extip_1.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
         <operations>
           <op name="monitor" interval="30s" id="extip_1_mon" timeout="30s"/>
         </operations>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="ip" value="85.125.191.221"/><nvpair id="nvpair" name="netmask" value="28"/><nvpair id="nvpair" name="nic" value="eth0"/><nvpair id="nvpair" name="target_role" value="started"/></instance_attributes>
       </primitive>
       <primitive id="extip_2" class="ocf" type="IPaddr2" provider="heartbeat"><meta_attributes id="primitive-extip_2.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
         <operations>
           <op name="monitor" interval="30s" id="extip_2_mon" timeout="30s"/>
         </operations>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="ip" value="85.125.191.222"/><nvpair id="nvpair" name="netmask" value="28"/><nvpair id="nvpair" name="nic" value="eth0"/><nvpair id="nvpair" name="target_role" value="started"/></instance_attributes>
       </primitive>
       <group id="group_main"><meta_attributes id="group-group_main.meta"/>
         <primitive id="intip_0_main" class="ocf" type="IPaddr2" provider="heartbeat"><meta_attributes id="primitive-intip_0_main.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
           <operations>
             <op name="monitor" interval="30s" id="intip_0_mon" timeout="30s"/>
           </operations>
           <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="ip" value="192.168.100.200"/><nvpair id="nvpair" name="netmask" value="24"/><nvpair id="nvpair" name="nic" value="eth1"/><nvpair id="nvpair" name="target_role" value="started"/></instance_attributes>
         </primitive>
       </group>
       <primitive id="intip_1_master" class="ocf" type="IPaddr2" provider="heartbeat"><meta_attributes id="primitive-intip_1_master.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
         <operations>
           <op name="monitor" interval="30s" id="intip_1_mon" timeout="30s"/>
         </operations>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="ip" value="192.168.100.201"/><nvpair id="nvpair" name="netmask" value="24"/><nvpair id="nvpair" name="nic" value="eth1"/><nvpair id="nvpair" name="target_role" value="started"/></instance_attributes>
       </primitive>
       <primitive id="intip_2_slave" class="ocf" type="IPaddr2" provider="heartbeat"><meta_attributes id="primitive-intip_2_slave.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
         <operations>
           <op name="monitor" interval="30s" id="intip_2_mon" timeout="30s"/>
         </operations>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="ip" value="192.168.100.202"/><nvpair id="nvpair" name="netmask" value="24"/><nvpair id="nvpair" name="nic" value="eth1"/><nvpair id="nvpair" name="target_role" value="started"/></instance_attributes>
       </primitive>
       <master id="ms_drbd_www"><meta_attributes id="master_slave-ms_drbd_www.meta"/><meta_attributes id="meta_attributes"><nvpair id="nvpair" name="master-max" value="2"/><nvpair id="nvpair" name="master-node-max" value="1"/><nvpair id="nvpair" name="clone-max" value="2"/><nvpair id="nvpair" name="clone-node-max" value="1"/><nvpair id="nvpair" name="target-role" value="started"/><nvpair id="nvpair" name="globally-unique" value="false"/><nvpair id="nvpair" name="notify" value="yes"/></meta_attributes><primitive id="drbd_www" class="ocf" provider="heartbeat" type="drbd"><meta_attributes id="primitive-drbd_www.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
           <operations>
             <op name="monitor" interval="5s" id="drbd_www_mon_normal" timeout="10s" role="Started"/>
           </operations>
           <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="drbd_resource" value="www"/><nvpair id="nvpair" name="clone_overrides_hostname" value="yes"/><nvpair id="nvpair" name="target_role" value="started"/></instance_attributes>
         </primitive></master>
       <clone id="clone_ocfs2_www"><meta_attributes id="clone-clone_ocfs2_www.meta"><nvpair id="nvpair" name="target-role" value="started"/><nvpair id="nvpair" name="clone-max" value="2"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><primitive id="ocfs2_www" class="ocf" provider="heartbeat" type="Filesystem"><meta_attributes id="primitive-ocfs2_www.meta"/>
           <operations>
             <op name="monitor" interval="120s" id="ocfs2_www_mon" timeout="60s"/>
           </operations>
           <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="device" value="/dev/drbd0"/><nvpair id="nvpair" name="directory" value="/data/www"/><nvpair id="nvpair" name="fstype" value="ocfs2"/></instance_attributes>
         </primitive></clone>
       <clone id="clone_webservice"><meta_attributes id="clone-clone_webservice.meta"><nvpair id="nvpair" name="target-role" value="started"/><nvpair id="nvpair" name="clone-max" value="2"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><group id="group_webservice"><meta_attributes id="group-group_webservice.meta"/>
           <primitive id="apache2" class="ocf" type="apache" provider="heartbeat"><meta_attributes id="primitive-apache2.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
             <operations>
               <op name="monitor" interval="60s" id="apache2_mon" timeout="300s"/>
             </operations>
             <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="httpd" value="/usr/sbin/apache2"/><nvpair id="nvpair" name="options" value="-k start"/><nvpair id="nvpair" name="configfile" value="/etc/apache2/apache2.conf"/><nvpair id="nvpair" name="target_role" value="started"/></instance_attributes>
           </primitive>
           <primitive id="mysql-proxy" class="lsb" type="mysql-proxy"><meta_attributes id="primitive-mysql-proxy.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
             <operations>
               <op name="monitor" interval="10s" id="mysql-proxy_mon" timeout="20s"/>
             </operations>
             <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="target_role" value="started"/></instance_attributes>
           </primitive>
         </group></clone>
       <master id="ms_drbd_mysql"><meta_attributes id="master_slave-ms_drbd_mysql.meta"/><meta_attributes id="meta_attributes"><nvpair id="nvpair" name="master-max" value="1"/><nvpair id="nvpair" name="master-node-max" value="1"/><nvpair id="nvpair" name="clone-max" value="2"/><nvpair id="nvpair" name="clone-node-max" value="1"/><nvpair id="nvpair" name="target-role" value="started"/><nvpair id="nvpair" name="globally-unique" value="false"/><nvpair id="nvpair" name="notify" value="yes"/></meta_attributes><primitive id="drbd_mysql" class="ocf" provider="heartbeat" type="drbd"><meta_attributes id="primitive-drbd_mysql.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
           <operations>
             <op name="monitor" interval="5s" id="drbd_mysql_mon_normal" timeout="10s" role="Started"/>
           </operations>
           <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="drbd_resource" value="mysql1"/><nvpair id="nvpair" name="clone_overrides_hostname" value="yes"/><nvpair id="nvpair" name="target_role" value="started"/></instance_attributes>
         </primitive></master>
       <primitive id="fs_mysql" class="ocf" provider="heartbeat" type="Filesystem"><meta_attributes id="primitive-fs_mysql.meta"/>
         <operations>
           <op name="monitor" interval="30s" id="fs_mysql_mon" timeout="30s"/>
         </operations>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="device" value="/dev/drbd1"/><nvpair id="nvpair" name="directory" value="/data/mysql"/><nvpair id="nvpair" name="fstype" value="ext3"/></instance_attributes>
       </primitive>
       <primitive id="mysql-server" class="ocf" provider="heartbeat" type="mysql"><meta_attributes id="primitive-mysql-server.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
         <operations>
           <op name="monitor" interval="30s" id="mysql-server_mon" timeout="30s"/>
         </operations>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="target_role" value="started"/><nvpair id="nvpair" name="priority" value="1000"/><nvpair id="nvpair" name="config" value="/etc/mysql/my.cnf"/><nvpair id="nvpair" name="socket" value="/var/run/mysqld/mysqld.sock"/><nvpair id="nvpair" name="pid" value="/var/run/mysqld/mysqld.pid"/><nvpair id="nvpair" name="datadir" value="/data/mysql/"/><nvpair id="nvpair" name="binary" value="/usr/sbin/mysqld"/><nvpair id="nvpair" name="additional_parameters" value="--basedir=/usr --skip-external-locking --port=42006"/></instance_attributes>
       </primitive>
     </resources>
     <constraints>
       <rsc_location id="run_extip_1" rsc="extip_1">
         <rule id="pref_run_extip_1" score="100">
           <expression attribute="#uname" operation="eq" value="webcluster01" id="expression"/>
         </rule>
       </rsc_location>
       <rsc_location id="run_extip_2" rsc="extip_2">
         <rule id="pref_run_extip_2" score="100">
           <expression attribute="#uname" operation="eq" value="webcluster02" id="expression"/>
         </rule>
       </rsc_location>
       <rsc_location id="run_intip_0_main" rsc="group_main">
         <rule id="pref_run_intip_0" score="100">
           <expression attribute="#uname" operation="eq" value="webcluster01" id="expression"/>
         </rule>
       </rsc_location>
       <rsc_location id="run_intip_1_master" rsc="intip_1_master">
         <rule id="pref_run_intip_1" score="200">
           <expression attribute="#uname" operation="eq" value="webcluster01" id="expression"/>
         </rule>
       </rsc_location>
       <rsc_location id="run_intip_2_slave" rsc="intip_2_slave">
         <rule id="pref_run_intip_2_wc2" score="100">
           <expression attribute="#uname" operation="eq" value="webcluster02" id="expression"/>
         </rule>
       </rsc_location>
       <rsc_order id="order_drbd_mysql_ip0" first-action="start" first="intip_1_master" then="ms_drbd_mysql" score="100"/>
       <rsc_order id="order_drbd_mysql_ip1" first-action="start" first="intip_2_slave" then="ms_drbd_mysql" score="100"/>
       <rsc_order id="order_drbd_www_ip0" first-action="start" first="intip_1_master" then="ms_drbd_www" score="100"/>
       <rsc_order id="order_drbd_www_ip1" first-action="start" first="intip_2_slave" then="ms_drbd_www" score="100"/>
       <rsc_colocation id="colo_drbd_mysql_ip0" rsc="drbd_mysql" with-rsc="intip_1_master" rsc-role="Master" score="100"/>
       <rsc_colocation id="colo_drbd_mysql_ip1" rsc="drbd_mysql" with-rsc="intip_2_slave" rsc-role="Slave" score="100"/>
       <rsc_colocation id="colo_drbd_www_ip0" rsc="drbd_www" with-rsc="intip_1_master" rsc-role="Master" score="100"/>
       <rsc_colocation id="colo_drbd_www_ip1" rsc="drbd_www" with-rsc="intip_2_slave" rsc-role="Slave" score="100"/>
       <rsc_order id="drbd_before_fs_mysql" first="ms_drbd_mysql" then="fs_mysql" then-action="start" first-action="promote" score="INFINITY"/>
       <rsc_colocation id="colo_drbd_fs_mysql" rsc="fs_mysql" with-rsc="ms_drbd_mysql" with-rsc-role="Master" score="INFINITY"/>
       <rsc_order id="drbd_before_ocfs2_www" first="ms_drbd_www" then="clone_ocfs2_www" then-action="start" first-action="promote"/>
       <rsc_colocation id="colo_ddrbd_ocfs2_www" rsc="clone_ocfs2_www" with-rsc="ms_drbd_www" with-rsc-role="Master" score="INFINITY"/>
       <rsc_order id="mysql-server_after_fs_mysql" first="fs_mysql" then="mysql-server" then-action="start" first-action="start"/>
       <rsc_colocation id="colo_mysql-server_and_fs_mysql" rsc="mysql-server" with-rsc="fs_mysql" with-rsc-role="Started" score="INFINITY"/>
       <rsc_order id="webservices_after_ocfs2_www" first="clone_ocfs2_www" then="clone_webservice" then-action="start" first-action="start"/>
       <rsc_colocation id="colo_webservices_and_ocfs2_www" rsc="clone_webservice" with-rsc="clone_ocfs2_www" with-rsc-role="Started" score="INFINITY"/>
       <rsc_colocation id="colocate_mysql-server_ip0" rsc="intip_0_main" with-rsc="mysql-server" with-rsc-role="Started" score="INFINITY"/>
     </constraints>
   </configuration>
   <status>
     <node_state id="49e81295-8e2f-4aeb-98f3-a14de6f62298" uname="webcluster01" crmd="online" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <lrm id="lrm.auto-1">
         <lrm_resources id="lrm_resources"/>
       </lrm>
     </node_state>
   </status>
 </cib>
